pkg/modules/generators/app_configurations_generator.go

```diff
@@ -1,33 +1,38 @@
 package generators
 
 import (
+	"errors"
 	"fmt"
 
 	"kusionstack.io/kusion/pkg/apis/intent"
 	"kusionstack.io/kusion/pkg/apis/project"
 	"kusionstack.io/kusion/pkg/apis/stack"
+	apisworkspace "kusionstack.io/kusion/pkg/apis/workspace"
```

Comment:
I would suggest rename `apisworkspace` to `workspaceapi`

Reply:
I am used to add the directory one level up as prefix if there are packages with same name, to keep the same alias rule.
But `workspaceapi` seems clearer than `apisworkspace`, I'll update it.